evap/evaluation/models.py

```diff
@@ -1626,16 +1630,22 @@ def send_to_user(self, user, subject_params, body_params, use_cc, additional_cc_
         body_params['contact_email'] = settings.CONTACT_EMAIL
 
         subject = self.render_string(self.subject, subject_params)
-        body = self.render_string(self.body, body_params)
+        plain_body = self.render_string(self.plain_body, body_params)
 
-        mail = EmailMessage(
+        mail = EmailMultiAlternatives(
             subject=subject,
-            body=body,
+            body=plain_body,
             to=[user.email],
             cc=cc_addresses,
             bcc=[a[1] for a in settings.MANAGERS],
             headers={'Reply-To': settings.REPLY_TO_EMAIL})
 
+        if self.html_body:
+            rendered_html_email_body = self.render_string(self.html_body, body_params)
+            html_template_params = dict({'email_body': rendered_html_email_body, 'email_subject': subject}, **body_params)
+            wrapped_html_email_body = render_to_string('email_base.html', html_template_params)
+            mail.attach_alternative(wrapped_html_email_body, "text/html")
```

Comment:
The template might contain user-specific data that the user might also be able to set themselves. so, using `self.render_string`, which sets `autoescape=False` isn't correct here (it might work for the current test data, but I'd rather have correct escaping in place. I'd even add a test for that.).

To test this, insert a `<` into a user's name. This must be escaped to `&lt;` in `rendered_html_email_body` (which with the current code shouldn't happen, if I'm not mistaken). It may be a good idea to also rename render_string to indicate that it disables autoescaping.

Then, the `render_to_string` call must not escape html, as we want to use the tags already in the content. This seems to already be the case -- but why? I quickly tried to find why no escaping happens here, but I didn't find anything.

Reply:
Yep that is a good point. I change it so the html content parameters and the subject string that is inserted into the template get escaped. For now I just changed the `self.render_string` method to optionally take an `autoescape` boolean that defaults to `False`. I dont know if that is the solution you looked forward to, but I think its not necessary to rename the method if we leave it this way. Especially because no autoescape it still mostly used.
For the `render_to_string` method: I could not find anything yet either and it is pretty strange...